docs/spec/index.html

```diff
@@ -272,89 +272,75 @@ <h2>Terminology</h2>
   <section data-include="sections/api-spec.html" data-include-replace="true">
   </section>
 
-  <section class="informative">
-    <h3>Workflows</h3>
+  <section class="normative">
+    <h2>Workflows</h2>
     <p>
-      <em>Workflows</em> are a critical concept in this specification that
-      enable presenters to correlate individual
-      presentations. A workflow <em>definition</em> can be used to indicate that
-      presentations relate to the same
-      <em>type</em>
-      of workflow.
+      <em>Workflows</em> are a critical concept in this specification that enable presenters to correlate individual presentations. A <em>Workflow Definition</em> can be used to indicate that a presentation intends to meet a defined set of data requirements; a <em>Workflow Instance</em> indicates that a presentation should be considered in conjunction with other presentations. 
+      <br />
+      The data model for indicating Workflow identifiers on Traceable Presentations are defined in <a href="https://w3c-ccg.github.io/traceability-interop/draft/#presentations">https://w3c-ccg.github.io/traceability-interop/draft/#presentations</a>. 
+    </p>
+    <h3>Workflow Definitions</h3>
+    <p class="advisement">
+      A Workflow Definition <em>MUST</em> be defined by a URN which resolves to a workflow definition manifest file.
     </p>
     <p>
-      Workflow <em>instances</em> indicate that separate individual
-      presentations are part of the same
-      <em>instance</em> of a specified workflow <em>type</em>.
+      The workflow definition manifest YAML file defines:
+      <ul>
+        <li>`id` identifier to be referenced.</li>
+        <li>`title` a title of the workflow definition.</li>
+        <li>`description` indicates applicable circumstances and expected procedures for the workflow.</li>
+        <li>`tags` listing applicable industry verticals.</li>
+        <li>`credentials` listing verifiable credentials types which are required to attain the goals of the workflow.</li>
+        <li>`mermaid` diagram which contextualizes the involved workflow parties and their roles</li>
```

Comment:
Right. Mermaid isn't and shouldn't be the be all end all; more nuanced images should be supported as too. 
I suggest we keep mermaid as an element for now, since that is what we have on trace-vocab. And then add/replace it with a more generic `image` later.